### PR TITLE
2.22-RC release anouncement with illustrations

### PR DESCRIPTION
**Pull request by jodygarnett**

### CODE REVIEW

**Review by mprins — APPROVED** [review]
👍🏽

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+### Release candidate public testing and feedback
+
+Testing and providing feedback on releases is part of the open-source social contract. The development team (and their employers and customers) are responsible for sharing this great technology with you.
+
+*The collaborative part of open-source happens now - we ask you to test this release candidate in your environment and with your data. Try out the new features, double check if the documentation makes sense, and most importantly let us know!**
+
+*If you spot something that is incorrect or not working do not assume it is obvious and we will notice. We request and depend on your [email](https://geoserver.org/comm/) and [bug reports](https://geoserver.org/issues/) at this time. If you are working with [commercial support](https://geoserver.org/support/) your provider is expected to participate on your behalf.*
+
+Keeping GeoServer sustainable as a long term community commitment. If you are unable to contribute time, [sponsorship options](https://github.com/geoserver/geoserver/wiki/Sponsor) are available via OSGeo.
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:29]: The first sentence doesn't make sense to me.

Possibly
```suggestion
Keeping GeoServer sustainable requires a long term community commitment. If you are unable to contribute time, [sponsorship options](https://github.com/geoserver/geoserver/wiki/Sponsor) are available via OSGeo.
```
?

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+### Natural Earth GeoPackage and workspace
+
+The sample [data directory](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22-RC/geoserver-2.22-RC-data.zip/download) now includes a small geopackage generated from [Natural Earth](https://www.naturalearthdata.com) data. These layers are a good example with multiple styles, and complete descriptions from the natural earth project.
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:33]:
```suggestion
The sample [data directory](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22-RC/geoserver-2.22-RC-data.zip/download) now includes a small geopackage generated from [Natural Earth](https://www.naturalearthdata.com) data. These layers are good examples with multiple styles, and include complete descriptions from the Natural Earth project.
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+![World map](/img/posts/2.22/world-map-rc.png) <br/>
+
+Thanks to Jody Garnett (GeoCat) and for the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:37]:
```suggestion
Thanks to Jody Garnett (GeoCat) and to the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+### User Manual Getting Started updated
+
+The user manual has been revised with:
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:41]:
```suggestion
The user manual has been revised. Changes include:
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+* [Getting started](https://docs.geoserver.org/latest/en/user/gettingstarted/index.html) revised with new sections for geopackage, image, layer group and style.
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:43]:
```suggestion
* [Getting started](https://docs.geoserver.org/latest/en/user/gettingstarted/index.html) has been updated and includes with new sections for GeoPackage, image, layer group and style.
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+Thanks to Jody Garnett (GeoCat) and the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).
+
+### Welcome Page Updates
+
+**Layout**
+
+The welcome page description provides a summary of the workspaces and layers available to the current user.
+
+The header includes welcome message and a link to the organization providing the service.
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:55]:
```suggestion
The header includes a welcome message and a link to the organization providing the service.
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+Each web services is listed using the service title as a heading, followed by the service abstract as a description. The protocols provided by the service are displayed as blocks linking to web service URL. These are the URLs used to access the service in a desktop or web application.
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:57]:
```suggestion
Each web service is listed using the service title as a heading, followed by the service abstract as a description. The protocols provided by the service are displayed as blocks linking to the web service URL. These are the URLs used to access the service in a desktop or web application.
```

**Comment by aaime** [comment]
Some new features are missing, I'm following up with one more commit adding them.

**Comment by aaime** [comment]
Done, added a few paragraphs at the end of the announcement.

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+Thanks to Niels for for this work.
+
+### Significant improvements in raster rendering performance
+
+Raster rendering performance has increased significantly for two specific use cases:
+
+* GeoTIFF hyperspectral images, with hundreds of bands, and band interleaved structure 
+* Mosaicking hundreds of small images
+
+[Hyperspectral sensors](https://en.wikipedia.org/wiki/Hyperspectral_imaging) collect information at a very high spectral resolution, producing images with hundreds of bands. The typical pixel interleaved layout, where all the bands of a single pixel are stored toghether, is particuarly inefficient while rendering a false color image,
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:151]:
```suggestion
[Hyperspectral sensors](https://en.wikipedia.org/wiki/Hyperspectral_imaging) collect information at a very high spectral resolution, producing images with hundreds of bands. The typical pixel interleaved layout, where all the bands of a single pixel are stored together, is particularly inefficient while rendering a false color image,
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 9]
```diff
@@ -0,0 +1,268 @@
...
+where only three of them are used. A band interleaved, where each band is stored in a separate bank, is more efficient. GeoServer used to load
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:152]:
```suggestion
where only three of them are used. A band interleaved, where each band is stored in a separate bank, is more efficient. GeoServer previously loaded
```

File: _posts/2022-10-19-geoserver-2-22-RC-released.md [hunk 10]
```diff
@@ -0,0 +1,268 @@
...
+System Property | Description
+-------------- | -----------
+azure.reader.accountName | The Azure account name
+azure.reader.accountKey | The Azure account key
+azure.reader.container | The Azure container for the blobs
+azure.reader.prefix | The optional prefix folder for the blobs
+
+#### STAC datastore and mosaicking
+
+A new community module, [STAC datastore](https://docs.geoserver.org/latest/en/user/community/stac-datastore/index.html, allows connecting
```
Comment (bradh) [inline _posts/2022-10-19-geoserver-2-22-RC-released.md:184]:
```suggestion
A new community module, [STAC datastore](https://docs.geoserver.org/latest/en/user/community/stac-datastore/index.html, supports connecting
```

**Comment by aaime** [comment]
I've checked the suggestions, they were all obvious improvements, so committed them all. Should be ready to go, whenever the tests on the RC artifacts are done and artifacts are published. Thanks for putting this toghether!

**Review by aaime — APPROVED** [review]

**Pull request merged**